media-libs/opencv/metadata.xml

```diff
@@ -17,24 +17,28 @@
 	<use>
 		<flag name="contrib">Install user contributed scripts from opencv_contrib (Experimental!)</flag>
 		<flag name="cuda">Enable NVIDIA Cuda computations support (Experimental!)</flag>
-		<flag restrict="&gt;=media-libs/opencv-4.1.2" name="download">Enable download during cmake configure</flag>
+		<flag name="cudnn" restrict="&gt;=media-libs/opencv-4.8.1-r1">Enable support for nVidia cuDNN library</flag>
+		<flag name="download" restrict="&lt;=media-libs/opencv-4.8.1">Enable download during cmake configure</flag>
+		<flag name="non-free" restrict="&gt;=media-libs/opencv-4.8.1-r1">Enable non-free components</flag>
```

Comment:
Restricting is not necessary if the flags are not present in other versions with different meanings. If $otherversion does not have the flag, description is not shown anyway.